android/app/src/main/kotlin/com/cBreez/client/job/RedeemSwap.kt

```diff
@@ -0,0 +1,54 @@
+package com.cBreez.client.job
+
+import android.content.Context
+import breez_sdk.BlockingBreezServices
+import breez_sdk.BreezEvent
+import breez_sdk.SdkException
+import com.cBreez.client.BreezNotificationHelper.Companion.notifyChannel
+import com.cBreez.client.Constants
+import com.cBreez.client.ForegroundService
+import com.cBreez.client.R
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.Json
+import org.tinylog.kotlin.Logger
+
+@Serializable
+data class AddressTxsConfirmedRequest(
+    val address: String,
+)
+
+class RedeemSwapJob(
+    private val context: Context,
+    private val fgService: ForegroundService,
+    private val payload: String,
+) : SDKJob {
+    companion object {
+        private const val TAG = "RedeemSwapJob"
+    }
+
+    override fun start(breezSDK: BlockingBreezServices) {
+        val request = Json.decodeFromString(AddressTxsConfirmedRequest.serializer(), payload)
+
+        try {
+            breezSDK.rescanSwaps()
```

Comment:
This is a very heavy call as it goes over all historical addresses and I think we don't want to use it within the job that is resources and time limited.
We need to find a way to refresh this specific address information.

Reply:
Would this be a good solution? breez/breez-sdk/pull/823

Reply:
Call to `rescanSwaps` removed: e05f412412efe6cde7df5c3d6411861489dddf3c